care/emr/reports/renderer/template_engine.py

```
@@ -0,0 +1,164 @@
+    @staticmethod
+    def _filter_phone(value: str) -> str:
+        if not value:
+            return ""
+        phone = (
+            str(value)
+            .replace(" ", "")
+            .replace("-", "")
+            .replace("(", "")
+            .replace(")", "")
+        )
+        if phone.startswith("+91") and len(phone) >= 13:  # noqa: PLR2004
```

Ruff (0.14.6)

Unused `noqa` directive (non-enabled: `PLR2004`)

Remove unused `noqa` directive

(RUF100)